### DISCUSSION TITLE
We should add examples in the README

### DISCUSSION THREAD

**Discussion by sappenin** [XRPLF/xrpl4j#95] [Ideas]
I happened upon this [Java SDK for another project](https://github.com/harmony-one/harmonyj), and thought they did a decent job of providing examples in the README. We should consider doing the same thing.

Any opinions about which examples would be best for us to show? In general, we should point people to the ITs, but having 4 or 5 key examples would be useful.

**Comment by intelliot** [comment]
We should also add a link to https://xrpl.org/get-started-using-java.html in the README... I'll open a quick PR.

**Reply by intelliot** [reply]
https://github.com/XRPLF/xrpl4j/pull/112

**Comment by sappenin** [comment]
I've also added https://github.com/XRPLF/xrpl4j/issues/262 which is related to this discussion -- I'd like to better integrate the examples that currently live in this project's ITs with the links on [the new examples page](https://xrpl.org/code-samples.html) on xrpl.org.
